easel/test/widget/gridview_banner_for_draft_published_test.dart

```diff
@@ -28,8 +29,8 @@ void main() {
           await tester.setScreenSize();
           await tester.testAppForWidgetTesting(
             Scaffold(
-              body: ChangeNotifierProvider(
-                create: (ctx) => GetIt.I.get<CreatorHubViewModel>(),
+              body: ChangeNotifierProvider.value(
+                value: GetIt.I.get<CreatorHubViewModel>(),
```

Comment:
There is no need to mock the `Repository` instead mock `CreatorHubViewModel` itself